packages/shared/src/rpc/__tests__/utils.test.ts

```diff
@@ -0,0 +1,60 @@
+import { validateSwapAmount } from '../utils';
+
+const swappingEnv = {
+  minimumSwapAmounts: {
+    Ethereum: { ETH: 100000000000000000n, FLIP: 0n },
+  },
+  maximumSwapAmounts: {
+    Ethereum: { ETH: 1000000000000000000n, FLIP: null },
+  },
+} as any;
+
+describe(validateSwapAmount, () => {
```

Comment:
it would be great to add one integration test for the `getQuote` endpoint to make sure we dont remove that check by mistake and give quotes that will be wrong

Reply:
Added as well